src/coreclr/vm/eetwain.cpp

```diff
@@ -1160,7 +1160,7 @@ HRESULT EECodeManager::FixContextForEnC(PCONTEXT         pCtx,
             if (pNewVar->startOffset <= newMethodOffset &&
                 pNewVar->endOffset   >  newMethodOffset)
             {
-                newMethodVarsSorted[(int)varNumber] = *pNewVar;
+                newMethodVarsSorted[varNumber] = *pNewVar;
```

Comment:
Just realized - undoing this will return a bug we fixed: https://github.com/dotnet/runtime/pull/72255/files

Reply:
I see. That warrants a comment.